services/basic_types.proto

```diff
@@ -1353,6 +1368,13 @@ message ServiceEndpoint {
      * The port of the node
      */
     int32 port = 2;
+
+    /**
+	   * A node domain name
+     * This MUST be the fully qualified domain name of the node.
+     * This value MUST NOT be more than 253 characters.
+	   */
+    string domain_name = 3;
```

Comment:
You still have tabs for indentation here.
Please replace with spaces (probably need to update IDE settings to use spaces in protobuf files; the default for IntelliJ is tabs).
Also, don't remove the html tags; without them everything is compressed to a single line and becomes unreadable.
We may want to include the clarification I asked:
```suggestion
    /**
     * A node domain name, including host.<br/>
     * This MUST be the fully qualified domain name of the node.<br/>
     * This value MUST NOT be more than 253 characters.
     */
    string domain_name = 3;
```

Reply:
Updated.